Makefile

```diff
@@ -256,6 +256,7 @@ codegen: types swagger manifests $(GOPATH)/bin/mockery docs/fields.md docs/cli/a
 	go generate ./...
 	make --directory sdks/java generate
 	make --directory sdks/python generate
+	make --directory sdks/rust generate
```

Comment:
Will want to add the generated files to the `.gitattributes`, similar to the [existing generated SDKs](https://github.com/argoproj/argo-workflows/blob/82310dd459aa080754169d6a1667d30d9b7c75bf/.gitattributes#L1)